plugins/woocommerce/changelog/31670-remove-star-icon

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: performance
+
+CYS - Fix activeThemeHasMods undefined error.
```

Comment:
The contents of this changelog file seem off -- maybe a rebase gone wrong? 🤔

Reply:
I've copy pasted the issue template to fire the action because this issue was too old, and I hadn't updated automatically. hope next commit will be fixed